templates/server-service.yaml

```
@@ -15,6 +15,11 @@ metadata:
   annotations:
 {{ template "vault.service.annotations" .}}
 spec:
+  {{- if and (.Values.server.service.externalTrafficPolicy ) (print .Values.server.service.externalTrafficPolicy | lower | title | eq "Local"  )}}
+  externalTrafficPolicy: {{ .Values.server.service.externalTrafficPolicy | lower | title | quote }}
+  {{- else }}
+  externalTrafficPolicy: {{ default "Cluster" | quote }}
+  {{- end}}
```

I feel like it may be simpler to document the possible traffic policies in `values.yaml` and then do:

```suggestion
  {{- if .Values.server.service.externalTrafficPolicy  }}
  externalTrafficPolicy: {{ .Values.server.service.externalTrafficPolicy }}
  {{- end }}
```